devtools-html/debugger.html
# Fix #3729 - Ensure 'Reveal in Tree' always works, even if source doesn't change

Associated Issue: #3729

### Summary of Changes

* Adding `clearShownSource` to allow reveal in source tree to work
* Calling explicitly in only two spots so it's not a burden on the rest of the app

### Test Plan

1.  Open a few sources
2.  Toggle a few arrows in the source tree closed, so as to obscure the the path to sources
3.  Right-click a tab, select "Reveal in Tree", and the tree should open to the currently focused source
4.  Right-click the source's content/code area, select "Reveal in Tree", and the tree should open to the currently focused source


### Notes
I like this approach because it's very explicit and only happens in two spots.  Thus `clearShownSource` doesn't have side effects for other, unrelated tasks
